### PR TITLE
Popup!

### PR DESCRIPTION
**Pull request by kiramarston**
It pops up. It also has icons. I will fix the duration/distance display once those handlers are in.
Screenshot 2023-02-01 at 17 09 46

### CODE REVIEW

File: packages/frontend/src/components/PopUpWrapper.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    <FontAwesomeIcon icon="fa-solid fa-train-subway" /> Train:
+                    {` ${details.train} metres`}
+                </p>
+                <p>
+                    <FontAwesomeIcon icon="fa-solid fa-train-subway" /> Nearest Train Station:
+                    {` ${details.nearestTrainStation.address} metres`}
+                </p>
+                <p>
+                    <FontAwesomeIcon icon="fa-solid fa-bus" /> Bus:
+                    {` ${details.train} metres`}
```
Comment (ChrisPalmerAND) [inline packages/frontend/src/components/PopUpWrapper.js:39]: rendering train deets here!

File: packages/frontend/src/components/PopUpWrapper.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                    <FontAwesomeIcon icon="fa-solid fa-person-walking" /> Walking:
+                    {` ${details.driving.distance} metres / ${details.driving.duration} seconds`}
+                </p>
+                <p>
+                    <FontAwesomeIcon icon="fa-solid fa-train-subway" /> Train:
+                    {` ${details.train} metres`}
+                </p>
+                <p>
+                    <FontAwesomeIcon icon="fa-solid fa-train-subway" /> Nearest Train Station:
+                    {` ${details.nearestTrainStation.address} metres`}
```
Comment (ChrisPalmerAND) [inline packages/frontend/src/components/PopUpWrapper.js]: this has the address but doesn't have the value for metres - that deliberately left out until we have the utils?

File: packages/frontend/src/components/PopUpWrapper.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                </p>
+                <p>
+                    <FontAwesomeIcon icon="fa-solid fa-bus" /> Bus:
+                    {` ${details.train} metres`}
+                </p>
+                <p>
+                    <FontAwesomeIcon icon="fa-solid fa-bus" /> Nearest Bus Stop:
+                    {` ${details.nearestBusStop.address} metres`}
```
Comment (ChrisPalmerAND) [inline packages/frontend/src/components/PopUpWrapper.js]: same as nearest Train

Comment (ChrisPalmerAND) [reply packages/frontend/src/components/PopUpWrapper.js]: actually - we dont have that data - we just have the address, so we can take metres out!

File: packages/frontend/src/components/PopUpWrapper.js (hunk 2 above)
Comment (ChrisPalmerAND) [reply packages/frontend/src/components/PopUpWrapper.js]: actually - we dont have that data - we just have the address, so we can take metres out!

File: packages/frontend/src/App.js [hunk 4]
```diff
@@ -5,6 +5,17 @@ import { AppBarWrapper } from './components/AppBarWrapper';
 import { DrawerWrapper } from './components/DrawerWrapper';
 import { MainWrapper } from './components/MainWrapper';
 import { Map } from './components/Map';
+import { library } from '@fortawesome/fontawesome-svg-core';
```
Comment (a-lor-cab) [inline packages/frontend/src/App.js:8]: should move all the relatedicon code in the popup Wrapper

File: packages/frontend/src/components/PopUpWrapper.js (hunk 3 above)
Comment (a-lor-cab) [inline packages/frontend/src/components/PopUpWrapper.js]: 🤔

**Review by ChrisPalmerAND — APPROVED** [review]
Iconic work 🔥

**Pull request merged**